macos/App/Contents/Frameworks/SDL2.framework/Headers/SDL_system.h

```diff
@@ -290,7 +290,7 @@ extern DECLSPEC const wchar_t * SDLCALL SDL_WinRTGetFSPathUNICODE(SDL_WinRT_Path
 extern DECLSPEC const char * SDLCALL SDL_WinRTGetFSPathUTF8(SDL_WinRT_Path pathType);
 
 /**
- *  \brief Detects the device family of WinRT plattform on runtime
```

Comment:
The folder Frameworks contains dependencies that are copied from the source. Please do not change these files, as it will make upgrading dependencies harder.